src/components/application_manager/src/commands/mobile/put_file_request.cc

```diff
@@ -187,9 +199,27 @@ void PutFileRequest::Run() {
     return;
   }
   const std::string full_path = file_path + "/" + sync_file_name_;
-  UNUSED(full_path);
+  const size_t bin_data_size = binary_data.size();
+
+  if ((*message_)[strings::msg_params].keyExists(strings::crc32_check_sum)) {
+    LOG4CXX_DEBUG(logger_, "Binary Data Size:  " << bin_data_size);
```

Comment:
@AByzhynar make it tace

Reply:
@LuxoftAKutsan Fixed in 9e46887993433e22387c52b6995a39274581646b